### PR TITLE
Live Editor: Retain Device Mode Sizing on panelUpdate

### LINKED ISSUE #753: Live Editor: After updating widget, mobile view is reset to desktop
To recreate this issue:

* Open the Live Editor.
* Switch to mobile or tablet.
* Edit and update a widget.

Note how the Live Editor doesn't maintain the preview. The mobile or tablet icon will still show as selected but the preview will rever to desktop.

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve  #753

### CODE REVIEW

**Comment by Misplon** [comment]
Thanks, mate. Is this change working for you? It isn't working for me.

**Comment by AlexGStapleton** [comment]
How does that look? Please note that you'll likely need to run `gulp build:dev` prior to doing a full build.

**Comment by Misplon** [comment]
Ahh ok full build.

**Comment by Misplon** [comment]
Sweet, working for me now. Nice.

File: js/siteorigin-panels/view/live-editor.js [hunk 1]
```diff
@@ -345,13 +345,18 @@ module.exports = Backbone.View.extend( {
...
+				if ( $( '.live-editor-mode.so-active' ).length ) {
+					$( '.so-panels-live-editor .so-preview iframe' ).css( 'transition', 'none' );
+					thisView.mobileToggle();
+				}
+
 				setTimeout( function(){
 					// Scroll to the correct position
 					$iframeContents.scrollTop( thisView.previewScrollTop );
 					thisView.$( '.so-preview-overlay' ).hide();
+					$( '.so-panels-live-editor .so-preview iframe' ).css( 'transition', 'all .2s ease');
```
Comment (Misplon) [inline js/siteorigin-panels/view/live-editor.js:357]: Just needs a space here after `ease`.

**Pull request merged**